test/python/test_convert.py

```diff
@@ -1320,21 +1330,20 @@ def checkConversion(self, basename, cls=ct.Solution, ctmlphases=(),
         ctmlPhase = cls(f"{basename}-from-xml.yaml", adjacent=ctmlphases, **kwargs)
         yamlPhase = cls(f"{basename}.yaml", adjacent=yamlphases, **kwargs)
 
-        self.assertEqual(ctmlPhase.element_names, yamlPhase.element_names)
-        self.assertEqual(ctmlPhase.species_names, yamlPhase.species_names)
-        self.assertEqual(ctmlPhase.n_reactions, yamlPhase.n_reactions)
+        assert ctmlPhase.element_names == yamlPhase.element_names
+        assert ctmlPhase.species_names == yamlPhase.species_names
+        assert ctmlPhase.n_reactions == yamlPhase.n_reactions
         for C, Y in zip(ctmlPhase.species(), yamlPhase.species()):
-            self.assertEqual(C.composition, Y.composition)
+            assert C.composition == Y.composition
 
         for C, Y in zip(ctmlPhase.reactions(), yamlPhase.reactions()):
-            self.assertEqual(C.__class__, Y.__class__)
-            self.assertEqual(C.reactants, Y.reactants)
-            self.assertEqual(C.products, Y.products)
-            self.assertEqual(C.duplicate, Y.duplicate)
+            assert C.__class__ == Y.__class__
+            assert C.reactants == Y.reactants
+            assert C.products == Y.products
+            assert C.duplicate == Y.duplicate
 
         for i, sp in zip(range(ctmlPhase.n_reactions), ctmlPhase.kinetics_species_names):
-            self.assertEqual(ctmlPhase.reactant_stoich_coeff(sp, i),
-                             yamlPhase.reactant_stoich_coeff(sp, i))
+            assert ctmlPhase.reactant_stoich_coeff(sp, i) == yamlPhase.reactant_stoich_coeff(sp, i)
```

Comment:
Please do wrap these overly long lines. I think my best suggestion here is
```suggestion
            assert (ctmlPhase.reactant_stoich_coeff(sp, i)
                    == yamlPhase.reactant_stoich_coeff(sp, i))
```